sources/packages/web/src/components/common/AwardTable.vue

```diff
@@ -55,13 +59,26 @@ export default defineComponent({
       if (awardValue === null) {
         return "-";
       }
+      if (awardValue) {
+        return (
+          "$" + awardValue.toLocaleString("en-CA", { minimumFractionDigits: 2 })
+        );
+      }
       // If the award in not defined at all it means that the award is not eligible and it was not
       // part of the disbursement calculations output.
-      return awardValue ?? "(Not eligible)";
+      return "(Not eligible)";
+    };
+
+    const getAwardType = (awardType: string): string => {
+      if (props.offeringIntensity === OfferingIntensity.partTime) {
+        return PartTimeAwardTypesObject[awardType];
+      }
+      return awardType;
```

Comment:
I totally acknowledge that this solution works. 

But at the same time I would like to suggest an alternative for your and team consideration. 

The existing `AwardDetail` object can have an additional property for display purpose.

STEP 1: 

![image](https://github.com/user-attachments/assets/ee5d901f-7f29-4b31-8c2e-0e09c1d06a3c)

STEP 2: 

Populate the `awardTypeDisplay` for PT with new values and for FT keep the `awardTypeDisplay` same as `awardType`

![image](https://github.com/user-attachments/assets/23c00f49-7100-4a37-bf66-4dede9972823)

STEP 3: 

![image](https://github.com/user-attachments/assets/1a0f8f67-516f-429d-85b0-e1c9f1fcce1a)


I see the benefit as avoiding additional mapping and also the if condition. Let me know your thoughts.

Reply:
I think this solution is better. I'll make the change.

Reply:
Updated.